src/containers/TexterTodoList.jsx

```diff
@@ -4,16 +4,57 @@ import CheckCircleIcon from "@material-ui/icons/CheckCircle";
 import Empty from "../components/Empty";
 import LoadingIndicator from "../components/LoadingIndicator";
 import AssignmentSummary from "../components/AssignmentSummary";
+import Snackbar from "@material-ui/core/Snackbar";
 import loadData from "./hoc/load-data";
 import gql from "graphql-tag";
 import { withRouter } from "react-router";
 
 let refreshOnReturn = false;
 
+// We want to back off on notifications both because if they don't leave the page
```

Comment:
This file, TexterTodoList is where all the action is on the client-side.  The changes do the following:
* separate the fields needed for an assignment record, so we can poll the same fields for notifications
* hard-code all the stupid count parameters so it's easy to embed them rather than pass them as variables
* back off notifications as a user continues to ignore them (if they click into a page and come back, it resets)